src/js/core/widget/core/Spin.js

```diff
@@ -677,13 +678,26 @@
 				utilsEvents.off(self.dragTarget, "drag dragend dragstart", self);
 			} else {
 				if (self.options.enabled) {
-					self._objectValue.value = this._startDragCount - e.detail.deltaY / DRAG_STEP_TO_VALUE;
+					self._objectValue.value = self._startDragCount - e.detail.deltaY / DRAG_STEP_TO_VALUE;
 					if (self.options.loop !== "enabled") {
 						self._objectValue.value = Math.min(Math.max(self._objectValue.value, 0), self.length - 1);
 					}
 					showAnimationTick(self);
 				}
 			}
+			// set timeout in case of drag outside screen
+			window.clearTimeout(self._dragTimeoutHandler);
+			self._dragTimeoutHandler = window.setTimeout(function () {
+				self._dragEnd({
+					detail: {
+						velocityY: e.velocityY,
+						distance: e.distance,
+						direction: e.direction
+					}
+				});
+				self._dragTimeoutHandler = null;
+				ns.event.gesture.Manager.getInstance().resetDetecting();
+			}, 500);
```

Comment:
ow about definig this 500 timeout as const together with other const?
see:
ENABLING_DURATION = 300, // [ms]
			ROLL_DURATION = 600,
			ROLL_DURATION = 600,
			DELTA_Y = 100,
			DELTA_Y = 100,
			DRAG_STEP_TO_VALUE = 60,
			DRAG_STEP_TO_VALUE = 60,
			NUMBER_OF_CAROUSEL_ITEMS = 11,
			NUMBER_OF_CAROUSEL_ITEMS = 13,

Reply:
ok. Done